src/Navigation.js

```diff
@@ -1,131 +1,61 @@
-                    </Nav.Item>
-                    {!isAuth ?
-                        <>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/register" eventKey="0">Register</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/login" eventKey="0" onClick={login}>Login</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                        </>
-                        :
-                        <Nav.Item>
-                            <Card>
-                                <Card.Header>
-                                    <Accordion.Toggle as={Link} to="/" eventKey="0" onClick={logout}>Logout</Accordion.Toggle>
-                                </Card.Header>
-                            </Card>
-                        </Nav.Item>
-                    }
-                    {isAuth && <Nav.Item>
-                        <Card>
-                            <Card.Header>
-                                <Accordion.Toggle as={Link} to="/" eventKey="0">Welcome back, {user}!</Accordion.Toggle>
-                            </Card.Header>
-                        </Card>
-                    </Nav.Item>}
+      <div className="topnav">
+               <Navbar collapseOnSelect className="ml-auto" expand="lg">
+                <div className="button-right">
+               <Navbar.Toggle  aria-controls='responsive-navbar-nav' />
+               <Navbar.Collapse id="responsive-navbar-nav">
+               <Nav className="ml-auto">
+                
+                      <Link to="/about" className="li">
+                         <li >About Us</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Services</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Membership</li>
+                      </Link>
+                      <Link to="/faq" className="li">
+                          <li>FAQs</li>
+                      </Link>
+                      <Link to="/aboutUs" className="li">
+                          <li>Blogs</li>
+                      </Link>
+                      <Link to="/getInTouch" className="li">
+                          <li>Contact Us</li>
+                      </Link>
+                      {!isAuth ?
+                      <>
+                      <Accordion.Toggle as={Link} to="/login" eventKey="0" className="li" onClick={login}>
```

Comment:
Also, you can use the same `Link` tag instead of `Accordion.Toggle` here. `eventKey` attribute is also unnecessary.

Reply:
I have made the suggested changes. Thanks, @mtreacy002 for the help :smile: